### PR TITLE
fixes line-height on userInfo-desc

### PR DESCRIPTION
**Pull request by saleiva**

### CODE REVIEW

**Comment by saleiva** [comment]
@CartoDB/frontend

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3408/))

**Comment by viddo** [comment]
OK for me

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3410/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3413/))

**Comment by saleiva** [comment]
Bump @CartoDB/frontend for CR, I've added a bunch of things

File: app/assets/stylesheets/public_map/public_map.css.scss [hunk 1]
```diff
@@ -54,10 +63,11 @@
 }
 .PublicMap-meta {
   display: inline-block;
-  margin-top: 10px;
+  margin-top: 4px;
   vertical-align: top;
+  font-weight: $sFontWeight-lighter;
 }
-.PublicMap-metaTime { @include flex-wrap(wrap) }
+.PublicMap-metaTime { @include flex-wrap(wrap); font-size: 15px; }
```
Comment (xavijam) [inline app/assets/stylesheets/public_map/public_map.css.scss]: If there is more than one rule, we don't recommend to have them in the same line.

File: app/assets/stylesheets/public_map/public_map.css.scss [hunk 2]
```diff
@@ -219,10 +230,13 @@
     width: auto;
     text-align: left;
   }
+  .PublicMap-buttons{
```
Comment (xavijam) [inline app/assets/stylesheets/public_map/public_map.css.scss]: Why?

File: app/assets/stylesheets/new_public_dashboard/user-info.css.scss [hunk 3]
```diff
@@ -44,6 +44,7 @@ $sAvatar-publicHalf: $sAvatar-public / 2;
 .UserInfo-desc {
   font-size: 18px;
   font-weight: $sFontWeight-lighter;
+  line-height: $sLineHeight-large;
```
Comment (xavijam) [inline app/assets/stylesheets/new_public_dashboard/user-info.css.scss]: :+1:

File: app/assets/stylesheets/public_map/public_map.css.scss (hunk 2 above)
Comment (saleiva) [inline app/assets/stylesheets/public_map/public_map.css.scss]: Not possible to edit the map on mobile or tablets plus those came with the share options in built in in the browser

Comment (xavijam) [inline app/assets/stylesheets/public_map/public_map.css.scss]: :+1:

**Comment by xavijam** [comment]
For me :+1:

File: app/assets/stylesheets/public_map/public_map.css.scss (hunk 1 above)
Comment (saleiva) [inline app/assets/stylesheets/public_map/public_map.css.scss]: Ok, fixed

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3415/))

**Comment by saleiva** [comment]
passtests!

**Comment by saleiva** [comment]
retest this please

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3420/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3421/))

**Pull request merged**